src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -220,12 +220,6 @@ public override void VisitNamedType(INamedTypeSymbol symbol)
             AddNameAndTypeArgumentsOrParameters(symbol);
         }
 
-        private static bool IsNullableType(INamedTypeSymbol type)
-        {
-            var original = type.OriginalDefinition;
-            return original != null && original.SpecialType == SpecialType.System_Nullable_T;
```

Comment:
:memo: this code lost null handling in the change

Reply:
> this code lost null handling in the change

This is intentional, ```type.OriginalDefinition```  is never null